src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -4834,6 +4834,15 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="ERR_RefReturningCallAndAwait" xml:space="preserve">
     <value>'await' cannot be used in an expression containing a call to '{0}' because it returns by reference</value>
   </data>
+  <data name="ERR_ByRefConditionalAndAwait" xml:space="preserve">
+    <value>'await' cannot be used in an expression containing a ref conditional operator</value>
+  </data>
+  <data name="ERR_RefConditionalNeedsTwoRefs" xml:space="preserve">
+    <value>Both or none of conditional operator expressions can be ref values</value>
```

Comment:
Both conditional operator values must be ref values or neither may be a ref value

---
In reply to: [105257393](https://github.com/dotnet/roslyn/pull/17436#discussion_r105257393) [](ancestors = 105257393)